source/pacboy/pacboy.sh

```diff
@@ -13,22 +13,17 @@ help_and_exit() {
         $(basename "$0") [command] [arguments]
         Arguments will be passed to pacman after translation:
 
-        For 64-bit MSYS2 shell:
-            name:i means i686-only
-            name:x means x86_64-only
-            name:z means clang-i686-only
-            name:c means clang-x86_64-only
-            name:u means ucrt-x86_64-only
-            name:a means clang-aarch64-only
-            name:p means MINGW_PACKAGE_PREFIX-only
-
-        For MSYS shell:
-            name:m means mingw-w64
-            name:l means mingw-w64-clang
-
-        For all shells:
-            name: disables any translation for name
-            repository::name means repository/name
+        Package name translations:
+            name:p - uses the package prefix of the current environment: ${full_package_prefix}name
+            name:i - translates to mingw-w64-i686-name
+            name:x - translates to mingw-w64-x86_64-name
+            name:z - translates to mingw-w64-clang-i686-name
+            name:c - translates to mingw-w64-clang-x86_64-name
+            name:u - translates to mingw-w64-ucrt-x86_64-name
+            name:a - translates to mingw-w64-clang-aarch64-name
+            name   - means the same as \"name:p\"
+            name:  - disables any translation for name
+            repository::name - means repository/name
```

Comment:
Without reading the rest of the script, can I combine this with the other translations? If I was using a repo that had multiple arch types (like the staging one) and wanted a specific clang-x86_64, could I specify `staging::qwt-qt5:c`? Either way, could this be mentioned if it can or cannot be combined with the above?

Reply:
looks like this works too. I've added a note